### PR TITLE
Feature/end game view

### PR DESCRIPTION
**Pull request by pkol92**
Add end game code and fix default timer

### CODE REVIEW

File: src/ts/App/App.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default function App(): void {
     const controller = new BoardController(root);
     const revertBtn = document.createElement('button');
     revertBtn.addEventListener('click', controller.undoMove);
-    revertBtn.addEventListener('click', (e)=>{
+    revertBtn.addEventListener('click', ()=>{
         const lastRecord = document.getElementById('last__move__info')?.lastChild;
         console.log(lastRecord)
```
Comment (kacus) [inline src/ts/App/App.ts:12]: leftover consol log

File: src/ts/App/Controllers/boardController.ts [hunk 2]
```diff
@@ -390,7 +392,15 @@ export default class BoardController {
...
-    if (+process.env.DEBUG!) console.log(`NEW GAME WILL START IN 5 SEC`);
-  }
+    let winerColor = '';
+    if (winer === 'b') {
+      winerColor = "BLACK"
+      } else {
+      winerColor = "WHITE"
+    };
+    const root = document.getElementById('root');
+    const end = new EndGame(root!);
```
Comment (kacus) [inline src/ts/App/Controllers/boardController.ts:402]: You can set exclamation mark in declaration, so you can use variable without need to remember about this exclamation mark.

File: src/ts/App/Controllers/boardController.ts [hunk 3]
```diff
@@ -390,7 +392,15 @@ export default class BoardController {
...
+
+    if (+process.env.DEBUG!) end.createWiner(winerColor);
```
Comment (kacus) [inline src/ts/App/Controllers/boardController.ts:404]: I don't know what EndGame class is. Can you explain me is it a View? If, yes why we want use it only in debug mode?

File: src/ts/App/Views/menu.ts [hunk 4]
```diff
@@ -71,10 +70,10 @@ export default class MenuView implements IMenu {
         insert_name_second.setAttribute('maxlength', '20');
         insert_name_second.setAttribute('pattern', '[a-zA-Z0-9]{3,20}');
         second_player.append(black, insert_name_second);
-        insert_name_second.addEventListener('change', (e)=>{
+        insert_name_second.addEventListener('change', ()=>{
             const element:HTMLInputElement = document.getElementById('second__player')! as HTMLInputElement;
```
Comment (kacus) [inline src/ts/App/Views/menu.ts:74]: If you use 'expression as type' expression and assign it to variable there is no need to tell TS the type of this variable is the same. TS can figure it out.

**Review by kacus — CHANGES_REQUESTED** [review]

File: src/ts/App/App.ts (hunk 1 above)
Comment (pkol92) [reply src/ts/App/App.ts]: done

File: src/ts/App/Controllers/boardController.ts (hunk 2 above)
Comment (pkol92) [reply src/ts/App/Controllers/boardController.ts]: done

File: src/ts/App/Controllers/boardController.ts (hunk 3 above)
Comment (pkol92) [reply src/ts/App/Controllers/boardController.ts]: Ok I removed it

**Pull request merged**